pkg/graveler/ref/manager_test.go

```diff
@@ -1111,3 +1111,49 @@ func TestManager_IsTokenExpired(t *testing.T) {
 		t.Fatalf("expected token expired")
 	}
 }
+
+func TestManager_DeleteExpiredAddressTokens(t *testing.T) {
+	r, _ := testRefManager(t)
+	ctx := context.Background()
+	repository, err := r.CreateRepository(ctx, "repo1", graveler.Repository{
```

Comment:
Question - we create a real repository, but use fake ref manager? if the answer is yes - we should just initialize repository record filled with information and not call the real API to create a repo.

Reply:
we use real ref manager